docs/federation.md

```diff
@@ -2,83 +2,74 @@
 
 ## Introduction
 
-The DACCS Network (“Network”) is a federated network that consists of independently operated and maintained computing
-infrastructure (“Node”) that deploy the DACCS software stack and connect with one another under the framework laid out
+The Marble Network ("Network") is a federated network that consists of independently operated and maintained computing
+infrastructure ("Node") that deploy the Marble software stack and connect with one another under the framework laid out
```

Comment:
I think some form of "DACCS" and "birdhouse" references should be kept here. 

Given that only the network uses "Marble", and that everywhere else (code, docs, etc.) refers to DACCS/birdhouse stack (including even this repo's name), the mention "Marble software stack" does not really provide any reference to users and adds more confusion to this already complex infrastructure.

Reply:
I disagree actually. Other code/docs refer to DACCS or birdhouse inconsistently and the point of coming up with a name like "Marble" is to create a consistent name that we can use to refer to the product that we are creating.

Here is how we should think about this:

- Marble: the name of the product
- DACCS: the name of the organization developing the product
- birdhouse: the name of a piece of software used to deploy one part of the product (a single node)
- magpie, cowbird, finch, etc.: components pieces of the product

Reply:
> birdhouse: the name of a piece of software used to deploy one part of the product 

Exactly as you said, hence why "the Marble software stack" doesn't make sense IMO. The software is birdhouse, not Marble. 
I agree with using Marble when referring to "The Marble Network".

Reply:
> Exactly as you said, hence why "the Marble software stack" doesn't make sense IMO. The software is birdhouse, not Marble.

Birdhouse is used for many things (as we've discussed at length). Marble is only one use-case for the birdhouse-deploy scripts. We've discussed in the past that birdhouse-deploy is used for many projects, not just Marble.

If you would like another way to think about it: "the Marble software stack can be deployed using a specific configuration of the birdhouse-deploy software".

Reply:
I don't see why birdhouse-deploy being used by many projects makes its name invalid when referring to what actually gets deployed. Birdhouse is the stack being deployed, and each node as its own configuration that must respect a preset defined by the Marble Network.

Otherwise, it's like saying that a Python-based program "X" runs the "X programming language" instead of "running X with Python". Python being used by many other programs is irrelevant in how it is applied in the case of "X".

Reply:
Yes... 

We are "running Marble with birdhouse-deploy" in the same way that you can "run X with Python". Maybe in the future we'll run Marble with something else, maybe not. But the point is that Marble is a software product that is distinct from birdhouse. We do not need to specify the underlying mechanism for deployment here.

> What would be the most explicit is simply to have [birdhouse](https://github.com/bird-house). That way, readers don't have to guess which software stack we refer to, there is an actual link.

Yes, but not here. We're talking about the product in the abstract here. Documentation for deploying and hosting a node belongs elsewhere.

Reply:
This documentation can be adjusted in the future as necessary if the stack evolves. For the moment, and I expect it to be for a long time, birdhouse-deploy is what is running. 

The whole point of this kind of documentation is to guide new node maintainers that would be interested in joining the network, not us that already know the stack. The first question anyone new will have is where this "Marble stack" is, so they can try it out. And then we would point them at birdhouse-deploy... instead of directly giving them the reference here right away. 

It's not like we are trying to form a secret organization. Why all this abstract mystery? Why not openly provide the link?

Reply:
I feel that the phrase "Marble software stack" makes sense because the stack can be based on something other than birdhouse in the future as pointed out by Misha. At present, the loose naming conventions from the past and the way the development was coordinated, have created an unfortunate situation that birdhouse, PAVICS, and now, DACCS/Marble have all come to mean vaguely similar things even though they are not. But we now have the opportunity to clarify the distinction between Marble and birdhouse and we should use that opportunity. 

Perhaps, we can add somewhere in the documentation that the "Marble software stack is currently using the community birdhouse deploy mechanism blah blah blah"?

Reply:
> The first question anyone new will have is where this "Marble stack" is, so they can try it out. And then we would point them at birdhouse-deploy... instead of directly giving them the reference here right away.

I think the website, that is due to go live soon, is going to have clear instructions on this. We can even add a link to that web page in this document.